partner_deduplicate_acl/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     "name": "Deduplicate Contacts ACL",
     "summary": "Contact deduplication with fine-grained permission control",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.0",
```

Comment:
You have changed the module version in the "pre-commit auto-fixes" commit instead of the migration one. I know that you are receiving a pre-commit error when trying to commit this one, but that's why in the migration guide, it puts that you must put `--no-verify` in the command for avoiding it. At the end, it's just to follow strictly the migration guide. Please place this in the proper place. I'm being picky here because if not, reviewing by commit is not effective and error prone to not detect possible problems.

Reply:
Absolutely understandable, omw to change it.

Reply:
Change is done now, although now the contributors change is now on the pre-commit fixes, which wouldnt be properly, only way to actually fully fix this is to re-do it from zero, if you would say that was the thing to do let me know and Ill re-do it, Ty

Reply:
Don't worry, let's keep it this time this way, not being critical.